### ISSUE TITLE
Console prints some extended unicode glyphs as 'unknown'

### ISSUE THREAD

**Issue by Zloten** [mihnita/ansi-econsole#84]
Ex. Unicode Character “─” (U+2500) is printed as 'unknown char' glyph (empty rectangle □).
The standard console does print it.
Use 'Droid Sans Mono' containing this glyph.
For 'Consolas' it prints valid.

**Comment by mihnita** [comment]
Can you please clarify what:
> The standard console does print it.

This plugin does not change the standard console, only adds colors to it.
It is the exact same console that is used without the plugin installed.

If a glyph shows with one font, but not with another, then the most likely explanation is that the font does not have the glyph.

In this case it is very easy to check.

Go to  https://fonts.adobe.com/fonts/droid-sans-mono and in the "Sample Text" paste the character with the problem.
It shows as missing glyph:

![image](https://user-images.githubusercontent.com/1307726/194010538-6c9d1832-6b53-494e-8bd4-eb4f600161da.png)

**Comment by mihnita** [comment]
TLDR: the 'Droid Sans Mono' font does not seem to contain the desired glyph.

You can also try to uninstall this plugin and see if the glyph shows (using the exact same font that didn't work before).
